packages/components/src/components/editor/condition-builder/ConditionBuilder.tsx

```diff
@@ -0,0 +1,72 @@
+import { IvyIcons } from '@axonivy/ui-icons';
+import { ConditionGroup } from './ConditionGroup';
+import { ConditionBuilderProvider, useConditionBuilderContext, type ConditionBuilderProviderProps } from './ConditionBuilderContext';
+import { BasicSelect } from '@/components/common/select/select';
+import { Flex } from '@/components/common/flex/flex';
+import { Button } from '@/components/common/button/button';
+import { useEffect, useState } from 'react';
+import type { Browser } from '@/components/editor/browser/browser';
+
+interface ConditionBuilderProps {
+  onChange: (value: string) => void;
+}
+
+type ConditionBuilderHookProps = Omit<ConditionBuilderProviderProps, 'children'>;
+
+const conditionModes = [
+  { label: 'Basic Condition', value: 'basic-condition' },
+  { label: 'Nested Condition', value: 'nested-condition' },
+  { label: 'Always True', value: 'always-true' },
+  { label: 'Always False', value: 'always-false' }
+];
```

Comment:
best mark this as const (readonly)
```suggestion
const conditionModes = [
  { label: 'Basic Condition', value: 'basic-condition' },
  { label: 'Nested Condition', value: 'nested-condition' },
  { label: 'Always True', value: 'always-true' },
  { label: 'Always False', value: 'always-false' }
] as const;
```

Reply:
ah okey. could be improved maybe :) 
leave it for now :)